### ISSUE TITLE
Error occurs running the tool

### ISSUE THREAD

**Issue by LifengWang** [kaiseu/pat-data-processing#1]
command
`/usr/bin/python2.7 /home/alex/workspace/pat-data-processing/processing.py -p /home/alex/pat/spark2.0_1TB_r5/pat_spark2.0_1TB_r5`

Error call stack
```python
only pat_path is assigned, calculating BENCHMARK average utilization...

Traceback (most recent call last):
  File "/home/alex/workspace/pat-data-processing/processing.py", line 271, in <module>
    run()
  File "/home/alex/workspace/pat-data-processing/processing.py", line 156, in run
    cluster_avg = Cluster(pat_path).get_cluster_data_by_time([0], [0], save_raw)
  File "/home/alex/workspace/pat-data-processing/cluster.py", line 79, in get_cluster_data_by_time
    avg = avg.append(tmp_avg.loc[i].mean(axis=0), ignore_index=True)
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1328, in __getitem__
    return self._getitem_axis(key, axis=0)
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1551, in _getitem_axis
    self._has_valid_type(key, axis)
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1442, in _has_valid_type
    error()
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1429, in error
    (key, self.obj._get_axis_name(axis)))
KeyError: 'the label [0] is not in the [index]'
```

**Comment by LifengWang** [comment]
This issues is caused by that vmstat, perf, jvms modules haven't be implemented.

**Comment by kaiseu** [comment]
vmstat, perf, jvms is not implemented yet, you should delete these files in your PAT raw data files. By the way, change default when only $pat_path is assigned to using parallel processing through commit 83a88be.

**Comment by LifengWang** [comment]
This issues still exists with this commit.
The error call stack is as follows:
```pyhton
Traceback (most recent call last):
  File "/home/alex/workspace/pat-data-processing/processing.py", line 274, in <module>
    run()
  File "/home/alex/workspace/pat-data-processing/processing.py", line 156, in run
    cluster_avg = get_cluster_data_by_time(pat_path, [0], [0], save_raw)
  File "/home/alex/workspace/pat-data-processing/parallelprocessing.py", line 98, in get_cluster_data_by_time
    cluster_avg[attr] = get_cluster_attrib_data(pat_path, start, end, save_raw, attr)
  File "/home/alex/workspace/pat-data-processing/parallelprocessing.py", line 81, in get_cluster_attrib_data
    avg = avg.append(tmp_avg.loc[i].mean(axis=0), ignore_index=True)
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1328, in __getitem__
    return self._getitem_axis(key, axis=0)
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1551, in _getitem_axis
    self._has_valid_type(key, axis)
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1442, in _has_valid_type
    error()
  File "/usr/lib64/python2.7/site-packages/pandas/core/indexing.py", line 1429, in error
    (key, self.obj._get_axis_name(axis)))
KeyError: 'the label [0] is not in the [index]'
```

**Comment by kaiseu** [comment]
vmstat, perf, jvms is not implemented yet, will ignored through commit 4265b13d351bf41247ef948cf20547d0d495632e, please try again.

**Comment by LifengWang** [comment]
Fixed in commit 4265b13d351bf41247ef948cf20547d0d495632e. Close this issue.
